rdflib/plugins/sparql/parserutils.py

```diff
@@ -241,16 +228,14 @@ def postParse(self, instring, loc, tokenList):
                 res["service_string"] = service_string
 
         for t in tokenList:
-            if isinstance(t, ParamValue):
-                if t.isList:
-                    if t.name not in res:
-                        res[t.name] = plist()
-                    res[t.name].append(t.tokenList)
-                else:
-                    res[t.name] = t.tokenList
-                # res.append(t.tokenList)
-            # if isinstance(t,CompValue):
-            #    res.update(t)
+            if isinstance(t, ParseResults):
+                for i in t:
+                    if isinstance(i, ParamValue):
+                        if i.name not in res:
+                            res[i.name] = plist()
+                        res[i.name].append(i.tokenList)
+            elif isinstance(t, ParamValue):
+                res[t.name] = t.tokenList
```

Comment:
@veyndan could you explain the significance of this? It may be right but I can't say I entirely understand why, I could look into it more though and it may be right but would be good to understand why you changed it.

Reply:
Okay nevermind, I see now it is needed to accomodate the change from `ParamList` to `Group(Param(...))`

Reply:
Resolving.